Example CsvTicksLoader.loadAppleIncSeries crashes when parsing data

Exception in thread "main" java.time.format.DateTimeParseException: Text '2013-01-02' could not be parsed: Unable to obtain ZonedDateTime from TemporalAccessor: {},ISO resolved to 2013-01-02 of type java.time.format.Parsed
	at java.time.format.DateTimeFormatter.createError(DateTimeFormatter.java:1920)
	at java.time.format.DateTimeFormatter.parse(DateTimeFormatter.java:1855)
	at java.time.ZonedDateTime.parse(ZonedDateTime.java:597)
	at ta4jexamples.loaders.CsvTicksLoader.loadAppleIncSeries(CsvTicksLoader.java:60)
	at ta4jexamples.loaders.CsvTicksLoader.main(CsvTicksLoader.java:79)
Caused by: java.time.DateTimeException: Unable to obtain ZonedDateTime from TemporalAccessor: {},ISO resolved to 2013-01-02 of type java.time.format.Parsed
	at java.time.ZonedDateTime.from(ZonedDateTime.java:565)
	at java.time.format.Parsed.query(Parsed.java:226)
	at java.time.format.DateTimeFormatter.parse(DateTimeFormatter.java:1851)
	... 3 more
Caused by: java.time.DateTimeException: Unable to obtain ZoneId from TemporalAccessor: {},ISO resolved to 2013-01-02 of type java.time.format.Parsed
	at java.time.ZoneId.from(ZoneId.java:466)
	at java.time.ZonedDateTime.from(ZonedDateTime.java:553)
	... 5 more

--- a/ta4j-examples/src/main/java/ta4jexamples/loaders/CsvTicksLoader.java
+++ b/ta4j-examples/src/main/java/ta4jexamples/loaders/CsvTicksLoader.java
@@ -57,7 +57,7 @@ public class CsvTicksLoader {
         try {
             String[] line;
             while ((line = csvReader.readNext()) != null) {
-                ZonedDateTime date = ZonedDateTime.parse(line[0], DATE_FORMAT);
+                ZonedDateTime date = LocalDate.parse(line[0], DATE_FORMAT).atStartOfDay(ZoneId.systemDefault());
                 double open = Double.parseDouble(line[1]);
                 double high = Double.parseDouble(line[2]);
                 double low = Double.parseDouble(line[3]);

--- a/ta4j-examples/src/main/java/ta4jexamples/loaders/CsvTicksLoader.java
+++ b/ta4j-examples/src/main/java/ta4jexamples/loaders/CsvTicksLoader.java
@@ -35,6 +35,8 @@ import java.util.logging.Logger;
 
 import com.opencsv.CSVReader;
 import java.time.ZonedDateTime;
+import java.time.ZoneId;
+import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
 
 /**

